docs/swagger.yaml

```
@@ -1,4 +1,14 @@
 definitions:
+  dto.LoginDto:
+    properties:
+      email:
+        type: string
+      password:
+        type: string
+    required:
+    - email
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)